core/validation/initial_validation/schemas.py

```diff
@@ -13,9 +13,19 @@
     AuthorisationCheck,
     BasicCheck,
     ConflictingCheck,
+    SheetCheck,
 )
 
 TF_ROUND_4_INIT_VAL_SCHEMA = [
+    SheetCheck(
+        sheet="1 - Start Here",
+        error_message=(
+            "The data return template you are submitting is not valid. Please make sure you are submitting a valid "
+            "template for Towns Fund."
```

Comment:
```suggestion
            "template for Towns Fund. "
```

nit: could use a space after the period in all of these (sorry 🙈)

Reply:
Would it be better to be consistent as we're currently not doing this for every error message?

Reply:
for any error messages * I should say